user/best-practices-security.md

```diff
@@ -0,0 +1,62 @@
+---
+title: Best Practices in Securing Your Data
+layout: en
+permalink: /user/best-practices-security/
+---
+
+## Steps Travis CI takes to secure your data
+Travis CI takes a number of steps to ensure secure environment variables and tokens are obfuscated when displayed in the UI. Our [documentation about encryption keys](https://docs.travis-ci.com/user/encryption-keys/) outlines the build configuration we require to ensure this, however, once a VM is booted and tests are running, we have less control over what information utilities or add-ons are able to print to the VM’s standard output. 
+
+In order to prevent leaks made by these components, we automatically filter secure environment variables and tokens at runtime, effectively removing them from the build log. The string `[secure]` will replace potentially leaked secure environment variables and tokens that are longer than three characters. 
```

Comment:
We've switched from "Travis CI takes" to "we filter"

Also

> In order to prevent leaks made by these components, we automatically filter secure environment variables and tokens that are longer than three characters at runtime, effectively removing them from the build log and displaying the string `[secure]` instead. 

Does that read better?